srcpkgs/occt/template

```diff
@@ -19,6 +19,16 @@ distfiles="https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/
 checksum=655da7717dac3460a22a6a7ee68860c1da56da2fec9c380d8ac0ac0349d67676
 conflicts="oce>=0"
 
+build_options="tbb"
+desc_option_tbb="Enable multithreaded algorithms"
+
+case "$XBPS_TARGET_MACHINE" in
+	# platforms for which tbb can be built
+	i686*|x86_64*|armv7l*|aarch64*|ppc*)
+		build_options_default+=" tbb"
```

Comment:
i think you shouldn't make it a build option, because why would you want to disable it if you can have it...
but rather add configure_args+="  -DUSE_TBB=ON" and makedepends+=" tbb-devel" in this case

Reply:
@q66 has mentioned that packages shouldn't have different features for different archs, if there isn't anything obvious differentiating between them. Using a build option as a way to make platform support clear is a preferred solution.

Reply:
I haven't said that, not in this context anyway. This is not really a feature per se.

Reply:
Should I do it without a build option, then?

Reply:
also, i'm fairly sure tbb can be made to work everywhere, and the current template is just wrong

Reply:
the result is the same for tbb-unsupported archs anyway, no matter if you make it a build_option or not. So it's either "have it not at all on" or have it but with not tbb support